src/Applications/NCEP_Etc/NCEP_enkf/scripts/gmao/get_atmens_rst.pl

```diff
@@ -24,11 +24,12 @@
 # main program
 #-------------
 {
-    my ($atmens_date3_dir, $atmens_date_dir, $atmens_ebkg_dir);
+    my ($atmens_date3_dir, $atmens_date_dir, $atmens_ebkg_dir, $atmens_ebkglfo_dir);
     my ($atmens_ecbkg_dir, $atmens_erst_dir, $atmens_stat_dir);
     my ($dmlist, $ens, $hh3, $hms3, $label, $mem, $mfile, $mfile_new, $mm3);
     my ($pid, $pwd, $rdnperts_dates3_txt, $tarfile, $tarpath);
     my ($three_hr_sec, $ymd3, $yyyy3, @tarList, @tarListA);
+    my (@labels, $bkglfo_present, $elfo );
```

Comment:
I recognize `elfo` is just a variable name, but for consistency it may be better to use `ebkglfo`.  
Similarly, maybe replace  `bkglfo_present` with `ebkglfo_present`

Reply:
same as above